### PR TITLE
Migration/bend

### PR DESCRIPTION
**Pull request by rvilarl**
Bend is there!

### CODE REVIEW

File: src/bend.ts [hunk 1]
```diff
@@ -7,6 +7,29 @@
...
+
+export interface Phrase {
+  x?: number;
+  type: number;
+  text: string;
+  width?: number;
+  draw_width?: number;
+}
+
+export interface State {
```
Comment (0xfe) [inline src/bend.ts:19]: This can be inline.

File: src/bend.ts [hunk 2]
```diff
@@ -7,6 +7,29 @@
 import { Vex } from './vex';
 import { Flow } from './tables';
 import { Modifier } from './modifier';
+
+export interface Phrase {
```
Comment (0xfe) [inline src/bend.ts:11]: Better to call this `BendPhrase`

File: src/bend.ts [hunk 3]
```diff
@@ -134,7 +170,7 @@ export class Bend extends Modifier {
     let total_width = 0;
     for (let i = 0; i < this.phrase.length; ++i) {
       const bend = this.phrase[i];
-      if ('width' in bend) {
+      if (bend.width) {
```
Comment (0xfe) [inline src/bend.ts:173]: This is wrong because `bend.width` could be 0.

File: src/bend.ts (hunk 1 above)
Comment (rvilarl) [reply src/bend.ts:19]: ok

File: src/bend.ts (hunk 3 above)
Comment (rvilarl) [reply src/bend.ts:173]: ok, checked with `undefined`

File: src/bend.ts (hunk 2 above)
Comment (rvilarl) [reply src/bend.ts:11]: ok

**Comment by rvilarl** [comment]
ready!

**Comment by 0xfe** [comment]
Good stuff!

**Comment by rvilarl** [comment]
@0xfe anything to be done?

**Comment by 0xfe** [comment]
Sorry, I missed this. Looks good! Thanks. :-)

**Pull request merged**